Dockerfile

```
@@ -1,4 +1,4 @@
-FROM --platform=linux/amd64 gcr.io/distroless/static-debian11
+FROM --platform=linux/${ARCH} gcr.io/distroless/static-debian11
```

```suggestion
ARG ARCH=amd64
FROM --platform=linux/${ARCH} gcr.io/distroless/static-debian11
```

Use command something like below when building image,
```
docker build --build-arg ARCH=${ARCH} ...
```